scripts/searchSamadi.py

```diff
@@ -0,0 +1,20 @@
+import commands, sys
+
+stringToLookFor = "*v0.1.0+76X_HHAnalysis_2016-04-04.v0"
+output = commands.getoutput("search_SAMADhi.py sample -n %s"%stringToLookFor)
```

Comment:
it's in `script` so it's already in the CMSSW python path ?
but I spoke too soon, there would be some slight changes to the script needed for the option parsing